chart/compass/charts/postgresql/values.yaml

```diff
@@ -109,7 +109,7 @@ postgresqlUsername: postgres
 ## Create a database
 ## ref: https://github.com/bitnami/bitnami-docker-postgresql/blob/master/README.md#creating-a-database-on-first-run
 ##
-# postgresqlDatabase:
+postgresqlDatabase: postgres # Modified
```

Comment:
remove Modified comment

Reply:
I thought it would be easier that way to keep track of changes made to postgres chart in case we will have to update it.